drivers/volume/portworx/portworx.go

```diff
@@ -3711,7 +3711,7 @@ func (p *portworx) getCloudBackupRestoreSpec(
 	}
 	var restoreSpec *api.RestoreVolumeSpec
 	destStorageClass := storageClassMapping[sourceStorageClass]
-	if len(destStorageClass) == 0 {
+	if len(destStorageClass) == 0 && len(destinationNamespace) == 0 {
```

Comment:
To understand, why we need this check len(destinationNamespace) == 0 ?

Reply:
Basically, we want do not want to set the locator labels, if the namespace mapping is not set.